tests/tests/main.rs

```diff
@@ -243,6 +243,40 @@ fn run_with_crash() {
         .failure();
 }
 
+#[test]
+fn run_with_coverage() {
+    let project = project("run_with_coverage")
+        .with_fuzz()
+        .fuzz_target(
+            "with_coverage",
+            r#"
+                #![no_main]
+                use libfuzzer_sys::fuzz_target;
+
+                fuzz_target!(|data: &[u8]| {
+                    run_with_coverage::pass_fuzzing(data);
+                });
+            "#,
+        )
+        .build();
+
+    project
+        .cargo_fuzz()
+        .arg("run")
+        .arg("--coverage")
+        .arg("coverage_output")
+        .arg("with_coverage")
+        .arg("--")
+        .arg("-runs=100")
+        .assert()
+        .stderr(
+            predicate::str::contains("Done 100 runs").and(predicate::str::contains(format!(
+                "Raw coverage data saved in coverage_output.profraw"
+            ))),
```

Comment:
Can we also assert that the `coverage_output.profraw` file exists?

Reply:
Yes, that makes sense. I now added that check right after we generate that file (i.e. outside of the test).

Reply:
(And also to the test.)